src/escalus_bosh_gun.erl

```diff
@@ -52,8 +52,9 @@ init([Args]) ->
     process_flag(trap_exit, true),
     Port = proplists:get_value(port, Args, 5280),
     Host = proplists:get_value(host, Args, <<"localhost">>),
+    GunOpts = gun_options(Args),
```

Comment:
Why introduce new variable and use it once only?

Reply:
Why not :) This is actually a leftover after debugging, initially there was no `GunOpts` var. If you insist I can change it, on the other hand it doesn't hurt much and compiler will optimise it anyway.